src/main/java/uk/gov/hmcts/reform/em/annotation/rest/TagResource.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.em.annotation.rest;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.hmcts.reform.em.annotation.service.TagService;
+import uk.gov.hmcts.reform.em.annotation.service.dto.AnnotationDTO;
+import uk.gov.hmcts.reform.em.annotation.service.dto.TagDTO;
+
+import java.util.List;
+
+/**
+ * REST controller for managing Tags.
+ */
+@RestController
+@RequestMapping("/api")
+public class TagResource {
+    private final Logger log = LoggerFactory.getLogger(TagResource.class);
+
+    private final TagService tagService;
+
+    public TagResource(TagService tagService) {
+        this.tagService = tagService;
+    }
+
+    /**
+     * GET  /tags/:id : get the tags "createdBy" user.
+     *
+     * @param createdBy the id of the user whose tags to retrieve
+     * @return the ResponseEntity with status 200 (OK) and with body the annotationDTO, or with status 404 (Not Found)
+     */
+    @ApiOperation(value = "Get list of tags created by user", notes = "A GET request to retrieve a list of tags")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Success", response = AnnotationDTO.class),
+            @ApiResponse(code = 401, message = "Unauthorised"),
+            @ApiResponse(code = 403, message = "Forbidden"),
+            @ApiResponse(code = 404, message = "Not Found"),
+    })
+    @GetMapping("/tags/{createdBy}")
+    //@Timed
```

Comment:
are there no endpoints to add/remove tags?

Reply:
It was decided that removal of tags would be an admin task. Adding tags happens as part of the edit comment process.